src/FluentValidation/Internal/PropertyRule.cs

```diff
@@ -181,7 +176,6 @@ TProperty PropertyFunc(T instance)
 				}
 			}
 
-			var cascade = CascadeMode;

```

Comment:
This variable was here to prevent the CascadeMode delegate from being invoked in every iteration of the loop, which helps with performance particularly with large validation sets, so this change will need to be undone and reverted please.

Reply:
Ah yeah sorry, should have noticed that!

Reply:
I should probably add a comment explaining that, as it's not obvious from a glance!